src/pages/Dashboard/inputComponent.css

```diff
@@ -0,0 +1,45 @@
+.form-field {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-bottom: 2rem;
+}
+
+.form-field input {
+  width: 20vw;
+  padding: 10px;
```

Comment:
Can we make there be no padding on the top so that the label text is closer to the input? (see the image I sent you)